tests/test_packages/test_connections/test_gym/test_gym.py

```diff
@@ -178,8 +190,6 @@ async def test_send_close_negative(self):
             dialogue_reference=self.dialogues.new_self_initiated_dialogue_reference(),
         )
         msg.to = self.gym_address
-        dialogue = self.dialogues.update(msg)
```

Comment:
Something got lost here...

Reply:
Here you are trying to 1. create a wrong message, 2. add it in a dialogue, 3. send it to the connection, and 4. check the connection raises an error. 

With the new error system for dialogue, it is not possible to do 2. So the incorrect message is just created and sent to the connection.

Reply:
Here you are trying to 1. create a wrong message, 2. add it in a dialogue, 3. send it to the connection, and 4. check the connection raises an error.

With the new error system for dialogue, it is not possible to do 2. So the incorrect message is just created and sent to the connection.

Reply:
@5A11 sounds good. but then this should be commented here in code; the next person will not immediately see this.

Reply:
Done